targets/tests/test_topics.py

```diff
@@ -0,0 +1,46 @@
+from django.core.files.storage import default_storage
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from targets.tests.factories import TopicFactory
+from users.tests.factories import UserFactory
+
+
+class TopicTests(APITestCase):
+    AUTHENTICATION_ERROR_MESSAGE = 'Authentication credentials were not provided.'
+
+    @classmethod
+    def setUpClass(self):
+        self.topic1 = TopicFactory()
+        self.topic1.save()
+        self.topic2 = TopicFactory()
+        self.topic2.save()
+        self.url = reverse('topics-list')
+        self.user = UserFactory()
+        self.user.save()
```

Comment:
Here the same comment, you don't need to save the object in the database. Factory Boy lets you either create and save the object or just create an instance.